playbook/app/pb_kits/playbook/pb_typeahead/components/Option.jsx

```diff
@@ -21,12 +20,12 @@ const Option = (props: any) => {
             <User
                 align="left"
                 avatarUrl={imageUrl}
-                name={label}
+                name={props.label}
```

Comment:
The `data.label` isn't going to be correct if you use `getOptionLabel`, so just use `props.label` instead.